index.html

```diff
@@ -1,59 +1,284 @@
+              <li>2015</li>
+            </ul>
+            <p>
+              Experiment content creation feature that allows users to add an
+              existing story over the course of a day without spamming their
+              friends
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>Ruby on</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button class="see-project">See Project</button>
+          </div>
+          <div class="card">
+            <img src="./images/snapshot4.png" />
+          </div>
+        </article>
+      </section>
+
+      <section id="about">
+        <div>
+          <h2>About<br />Myself</h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <div class="social-media">
+            <a href="#" class="social-para social-blue">LET'S CONNECT</a>
+            <ul>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.github.com/joeveoper"
+                  ><img src="./images/twitter.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.linkedin.com/in/yinka-olatunji-83aa36b5/"
+                  ><img src="./images/linkedin.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.twitter.com/joeveoper"
+                  ><img src="./images/medium.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.medium.com/"
+                  ><img src="./images/github.png"
+                /></a>
+              </li>
+              <li>
+                <a
+                  target="_blank"
+                  class="social-tag"
+                  href="https://www.angellist.com/"
+                  ><img src="./images/angellist.png"
+                /></a>
+              </li>
+            </ul>
+            <button class="see-project">Get My Resume</button>
+          </div>
+          <h3>Languages</h3>
+          <ul class="about-skills">
+            <li><img src="./images/js.png" /> Javascript</li>
+            <li><img src="./images/html.png" /> HTML</li>
+            <li><img src="./images/css.png" /> CSS</li>
+          </ul>
+          <h3 class="under">Frameworks</h3>
+          <h3 class="under">Skills</h3>
```

Comment:
Nice job so far😉 we can also make this section better by sticking to the design😉 You have forgotten to add the arrows😉😉

yours
<img width="281" alt="arrows" src="https://user-images.githubusercontent.com/64643995/220054855-f566490f-ddbf-409b-bfd1-a16f16dd74ad.png">
design
<img width="238" alt="design" src="https://user-images.githubusercontent.com/64643995/220055051-3925e493-1ca5-4f2e-8ffd-95c8b76d0772.png">

kindly ensure you stick to the design😊